Week1/queries.js

```diff
@@ -0,0 +1,98 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect(error => {
+  if (error) {
+    throw error;
+  }
+  console.log('Connected.');
+});
+
+const queries = {
+  createDatabase: `CREATE DATABASE IF NOT EXISTS world;`,
+
+  useDatabase: `USE worldData`,
+
+  createTableCountry: `CREATE TABLE IF NOT EXISTS country (code CHAR(3), Name CHAR(52), Continent CHAR (30), Region CHAR(26),SurfaceArea FLOAT(10,2),Indepyear SMALLINT(6),Population INT(11),LifeExpectancy FLOAT(3.1),GNP FLOAT(10.2),LocalName CHAR(45),GovernmentForm CHAR(45),HeadOfState CHAR(60),Capital INT(11),Code2 CHAR(2),PRIMARY KEY(Code));`,
```

Comment:
I see you are using CHAR a lot. This is not wrong per se but it is good to understand the difference

https://stackoverflow.com/questions/1885630/whats-the-difference-between-varchar-and-char

Reply:
Actually the query was in the real sql file. Now I changed chars into varchars. Thanks!